client/src/components/Button/Button.css

```diff
@@ -0,0 +1,61 @@
+@import url("https://fonts.googleapis.com/css?family=Reem+Kufi&display=swap");
+
+.Button {
+  width: 200px;
+  height: 50px;
```

Comment:
Rather than setting explicit height and width values, can you assign padding ratios? That way, these will look consistent when the text is short and when it's long